lib/influxer/metrics/metrics.rb

```diff
@@ -162,10 +170,50 @@ def tags
       @attributes.select { |k, _| tag_names.include?(k.to_s) }
     end
 
-    attributes :timestamp
+    def time(parse = false)
```

Comment:
For this case (current time precision is 'ms'):
Pass a date time or time for time=.
Internally will be converted to an integer value by the method `parse_time`.
The time with flag parse this integer value to time instance.

Suggests some modification?